### PR TITLE
[KOD-321] Use enterprise version of metabase jar

### PR DESCRIPTION
**Pull request by surfacedamage**
## Description
Metabase has two installable versions of its product:  a free, open source version and an enterprise/pro version.  These are two completely separate jars, so you can't easily upgrade from the open source version to enterprise simply by entering a license key.  You need to swap out the jars first.

The primary change of this PR is to point the downloadable URL to the enterprise jar.

## References
KOD-321

## Types of changes
- ✅ Chore

### CODE REVIEW

File: bin/version [hunk 1]
```diff
@@ -1 +1 @@
-0.45.2.1
+1.45.2.1
```
Comment (saterus) [inline bin/version:1]: Well that's a rather large jump. Was that a typo?

**Review by saterus — APPROVED** [review]
> These are two completely separate jars, so you can't easily upgrade from the open source version to enterprise simply by entering a license key. You need to swap out the jars first.

Do we need to enter a license key *later*? Is there any restrictions on using the enterprise jar without that?

**Comment by surfacedamage** [comment]
@saterus Do we need to enter a license key later?

Yes, we enter the key after we upgrade.  Our current jar doesn't even give us the form to do it.  The license key is already purchased and in 1p.

File: bin/version (hunk 1 above)
Comment (surfacedamage) [reply bin/version:1]: Not a typo.  They keep the same semantic versioning between the enterprise and non-enterprise JARs, but the enterprise begins with `1.0xxx` and the non-enterprise begins with `0.0.xxx`.  🤷 

So, knowing that -- this diff'd line says, "same version, but switch to enterprise."

**Pull request merged**